### ISSUE TITLE
Fine tuning a v8 pose model appears to be using the wrong loss function, unclear how to define loss function

### ISSUE THREAD

**Issue by bs7280** [ultralytics/ultralytics#14742]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hello, 

Im trying to fine tune a pose model with my own annotations, where I have changed the position of a few keypoints to be more accurate (like shoulder position) but am getting no improvement after 2 epochs:

`EarlyStopping: Training stopped early as no improvement observed in last 100 epochs. Best results observed at epoch 2, best model saved as best.pt.` 

This makes me think that the pose model is trying to use the same loss function as the bounding box and detection, which is unchanged in my annotations. 

```
model = YOLO("yolov8n-pose.pt")
result = model.train(data=dataset_path, epochs=200)
```

I have also verified that my annotations are noticeably different than the default pose prediction, meaning there should be some sort of room for the model to improve, but when the mode is finished, the predictions are still incorrect.

When I try to set the loss function as an arg to model.train I get this:

```
*** SyntaxError: 'lf' is not a valid YOLO argument. Similar arguments are i.e. ['lrf=0.01', 'half=False'].
```

But I found this documentation that implies many loss functions are available? [loss - Ultralytics YOLO Docs](https://docs.ultralytics.com/reference/utils/loss/#ultralytics.utils.loss.v8PoseLoss)

Yet these training docs don't provide info on how to set the loss function:

[Configuration - Train Settings - Ultralytics YOLO Docs](https://docs.ultralytics.com/usage/cfg/#train-settings)

**TLDR - Fine tuning doesn't have any improvement after 2 epochs, but model still doesn't fit to my pose data, meaning loss function is probably wrong**

thanks in advance

**Comment by pderrenger** [comment]
@bs7280 thank you for reaching out. It sounds like you're encountering issues with fine-tuning your YOLOv8 pose model. The loss function for pose estimation is indeed different from that used for bounding box detection. The model should automatically use the appropriate loss function for pose tasks.

To address your issue, please ensure you are using the latest version of the Ultralytics package, as updates often include important fixes and improvements. If the problem persists, it might be helpful to verify that your annotations are correctly formatted and that the dataset is properly configured.

For further assistance, you can refer to the [YOLOv8 Common Issues Guide](https://docs.ultralytics.com/guides/yolo-common-issues/) for additional troubleshooting steps. If you continue to experience difficulties, please provide more details or any error messages you encounter, and we can look into it further.

**Comment by Y-T-G** [comment]
If it was using the wrong loss function, the code would error out.

You can try adjusting the loss weights by passing the new value as args:
https://github.com/ultralytics/ultralytics/blob/f8454f1ab71b860371deb03591342f6d6c87efac/ultralytics/cfg/default.yaml#L97-L101
